etc/spec/SPECIFICATION.md

```diff
@@ -146,43 +163,17 @@ follows:
 
 The error message is displayed above the textbox.
 
-![Simple login screen error](http://i.imgur.com/se80ce8.jpg)
-*Image: An error logging in using the simple login screen*
-
-Below the username textbox, a link allowing the user to login with their
-reserved username is displayed, entitled "Έχω ήδη ψευδώνυμο". Clicking that
-link changes the Simple Login screen into the Password Login screen.
-
-## Password Login
+![Login Screen/Error username](http://i.imgur.com/hXJThLI.jpg?1)
+*Image: An error logging in with invalid username*
 
-The Password Login screen allows the user to login with a username and a
-password of a reserved username. It is identical to the Simple Login screen,
-except it contains a password input underneath the username input and it
-doesn't contain a link at the bottom. The placeholders for the input boxes here
-are "Το ψευδώνυμό σου" for the username input and "Ο κωδικός σου" for the
-password input.
+The error message "Το ψευδώνυμο το έχει άλλος." appears only if someone is using
```

Comment:
Not sure when exactly this message will appear. I thought if a username is registered then the password form will appear. No?